app/assets/stylesheets/earthworks.css

```diff
@@ -208,10 +208,13 @@ Uncomment lines 142-144
 
 .expand-collapse .btn-outline-secondary {
```

Comment:
It's really odd that we're changing secondary buttons to be blue.  The component library defines "primary" as the blue color.  Is there a reason we can't change the class?